modules/host/host.go

```diff
@@ -38,31 +38,30 @@ type contractObligation struct {
 // A Host contains all the fields necessary for storing files for clients and
 // performing the storage proofs on the received files.
 type Host struct {
-	// modules
+	// Module dependencies.
 	cs     modules.ConsensusSet
 	tpool  modules.TransactionPool
 	wallet modules.Wallet
 
-	// resources
-	listener net.Listener
-	log      *log.Logger
-
-	// variables
-	blockHeight         types.BlockHeight
+	// File management.
+	fileCounter         int
 	obligationsByID     map[types.FileContractID]*contractObligation
 	obligationsByHeight map[types.BlockHeight][]*contractObligation
-	spaceRemaining      int64
-	fileCounter         int
 	profit              types.Currency
+	spaceRemaining      int64
+
+	// Persistent settings.
+	blockHeight types.BlockHeight
```

Comment:
might seem more relevant when the block height gets added to the persistence. This is a change-in-progress. Adding block height will actually require two fields, `height` and `recentChange`. This will allow the host to do more efficient subscription at startup (instead of decoding every block in the database, only the ones that the host has never seen before (likely none or only one) will need to be decoded and given to the host.

It's not a setting, and maybe belongs with the utilities or module dependencies, or maybe belongs by itself. I'm not going to change it in this PR, but if you still think it doesn't belong, I can have it and `recentChange` go somewhere else in the next one.